recipes/ignition-tools/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```

To be sure that Conan will not solve more things than provided by cmake_find_package_multi